### PR TITLE
updated README to correct errors

### PR DESCRIPTION
**Pull request by KeenanRileyFaulkner**
…ke branch for install

The use of make branch is temporary, until Reilly can update main branch makefiles to match

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,72 +4,86 @@
...
 ### Fasm2Bels
-1. Install capnproto_java:
-    * ```make capnproto_java```
-1. Install Rapidwright
-   * ```make rapidwright```
-   * ```make env```
-   * ```. ./env.sh```
-1. Install fasm2bels
-   * ```make install_fasm2bels```
+Information found here: https://github.com/chipsalliance/f4pga-xc-fasm2bels/blob/master/README.md
```
Comment (jgoeders) [inline README.md:77]: Can you add < > around the link?

File: README.md [hunk 2]
```diff
@@ -4,72 +4,86 @@
...
+    * This tool depends on Project X-Ray (also installed with bfasst): https://github.com/f4pga/prjxray/blob/master/README.md
 
 ### Yosys
```
Comment (jgoeders) [inline README.md:80]: Can you add a section like this for rapidwright?

File: README.md [hunk 3]
```diff
@@ -4,72 +4,86 @@
...
-1. Install Yosys
-    * ```make install_yosys```
+An open source synthesis tool. https://github.com/YosysHQ/yosys/blob/master/README.md
```
Comment (jgoeders) [inline README.md:81]: Add < > around link

**Review by jgoeders — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -4,72 +4,86 @@
...
+  --map MAP             Mapping args
+  --cmp CMP             Comparison args
+  --reverse REVERSE     Reverse args
+  --err ERR             Error flow args
+  --quiet
+  --error_flow {single_bit_flip,tap_signal,cross_wires}
+                        YAML file describing errors to inject for testing.Only works with flows designed for error injection
 ```
 
 There are also several pre-configured *experiments*, which allow you to run a large set of designs and collect results.  These configurations are located within the `experiments` directory, and can be run using `./scripts/run_experiment.py`:
```
[the remark this answers is not in the document]
Comment (KeenanRileyFaulkner) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -4,72 +4,86 @@
...
 ```
 
 ## Install
-
+### Prerequisites
+* Install Vivado 2022.2
+  * Update bfasst/config.py_ and make sure the path points to your Vivado installation. 
 * Clone this github repository. 
+* Checkout the make branch
+  * ```git checkout make```
```
[the remark this answers is not in the document]
Comment (KeenanRileyFaulkner) [reply README.md]: Done

File: README.md [hunk 6]
```diff
@@ -4,72 +4,86 @@
...
+* Install Capnproto Java
+  * ```sudo make capnproto_java```
+### Install BFASST
 * Install the Python virtual environment, activate, and install packages:
-  * ```make venv```
-  * ```. .venv/bin/activate```
-  * ```pip install -r requirements.txt```
+  * `make install`
+* Return to main branch
+  * ```git checkout main```
```
[the remark this answers is not in the document]
Comment (KeenanRileyFaulkner) [reply README.md:62]: Done

File: README.md (hunk 1 above)
Comment (KeenanRileyFaulkner) [reply README.md]: Done

File: README.md [hunk 7]
```diff
@@ -4,72 +4,86 @@
...
-1. Install capnproto_java:
-    * ```make capnproto_java```
-1. Install Rapidwright
-   * ```make rapidwright```
-   * ```make env```
-   * ```. ./env.sh```
-1. Install fasm2bels
-   * ```make install_fasm2bels```
+Information found here: https://github.com/chipsalliance/f4pga-xc-fasm2bels/blob/master/README.md
+    * This tool depends on Project X-Ray (also installed with bfasst): https://github.com/f4pga/prjxray/blob/master/README.md
```
[the remark this answers is not in the document]
Comment (KeenanRileyFaulkner) [reply README.md]: Done

**Comment by jgoeders** [comment]
@KeenanRileyFaulkner It looks like Reilly already updated the Makefile as part of #173.  Can you resolve this conflict and see which of your edits are still needed? Thanks!!

**Comment by KeenanRileyFaulkner** [comment]
I looked through the conflict and resolved it. The new README should have the correct path for /designs rather than /examples, <> around links, a section for RapidWright docs, and correct information on Fasm2Bels

**Review by jgoeders — CHANGES_REQUESTED** [review]

**Pull request merged**